content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -734,6 +735,15 @@ Each feature gate is designed for enabling/disabling a specific feature:
   See [setting kubelet parameters via a config file](/docs/tasks/administer-cluster/kubelet-config-file/)
   for more details.
 - `KubeletCredentialProviders`: Enable kubelet exec credential providers for image pull credentials.
+- `KubeletInUserNamespace`: Enables support for running kubelet in a user namespace.
+   The user namespace has to be created before running kubelet.
+   All the node components such as CRI need to be running in the same user namespace.
```

Comment:
For user documentation here would be helpful to say what would be result otherwise - how user will diagnose incorrect setting

Reply:
I'm not sure. I don't think a user may execute them in different user namespaces accidentally.

Reply:
From user's perspective, it is a little bit confusing how the chicken-egg problem is resolved. I have to have a namespace before running kubelet. How can that happen? This is only used when adding a new worker node to an existing cluster?

Reply:
This is not about placing Nodes in namespaces per the API (I think); it's about running the kubelet on Linux after setting `CLONE_NEWUSER` during process creation to set up a new user namespace on the host.

I believe that the kubelet also doesn't directly create any user namespace, though its chosen container runtime might.

Reply:
In our docs, let's help make sure this difference is clear to readers.

Reply:
Added `content/en/docs/reference/glossary/userns.md`